client/components/common/Functions.tsx

```diff
@@ -0,0 +1,3 @@
+export const capitalizeFirstLetter = (str: string): string => {
```

Comment:
woah, deja vu

This change also shows up in #204, I'm guessing that's because #204 depends on this to merge first?

Reply:
Yup